tasks/centos.yml

```diff
@@ -11,6 +11,7 @@
     - php{{ php_version | replace('.','') }}-php-pgsql
     - php{{ php_version | replace('.','') }}-php-pdo
     - php{{ php_version | replace('.','') }}-php-mbstring
+    - php{{ php_version | replace('.','') }}-php-xml
```

Comment:
I think it was for composer. Also required for another project.
If not required by composer, I will remove it.